301-eventHub-create-authrule-namespace-and-eventHub/README.md

```diff
@@ -0,0 +1,5 @@
+# 301-eventHub-create-authrule-namespace-and-eventHub
```

Comment:
This doesn't seem like a 301 template, more like 201. There's only a few resources in this.